js/views/modals/orderDetail/summaryTab/Summary.js

```diff
@@ -936,6 +935,10 @@ export default class extends BaseVw {
       });
     }
 
+    if (this.shouldShowCompleteOrderForm()) {
+      this.renderCompleteOrderForm();
+    }
```

Comment:
It might be better to move this after the sort. so it shows up at the top. Currently is gets added in the middle of the other sections, and you have to scroll down to see it.

![image](https://user-images.githubusercontent.com/1584275/43109498-17b6769c-8eb5-11e8-8d2a-8e321af60933.png)

![image](https://user-images.githubusercontent.com/1584275/43109481-05510d3c-8eb5-11e8-8e45-9bbfc7888e93.png)

Reply:
Yeah, good catch. I couldn't test properly because my buyer just wasn't getting the PAYMENT_FINALIZED message.